### ISSUE TITLE
Tabbed code blocks under admonitions and other elements

### ISSUE THREAD

**Issue by facelessuser** [squidfunk/mkdocs-material#796]
## Description

New tabbed code looks weird on mobile when under an admonition or details element.  I've noticed that that you have this style in the superfences.scss:

```scss
    // [mobile +]: Collapse tab labels
    @include break-from-device(tablet) {
      border: 0.1rem solid $md-color-black--lightest;
    }
```

I understand there is the same kind of logic for this as code blocks so that on mobile, it spreads to the width of the page, but in admonitions and details, there is an exception that prevents code blocks from doing this in them, but there doesn't seem to be the same exception for tabbed code blocks:

<img width="458" alt="2018-06-12 12_41_20-pymdown extensions documentation" src="https://user-images.githubusercontent.com/1055125/41310794-c67b8bea-6e3f-11e8-993d-cfa185073347.png">

### Expected behavior

Tabbed code blocks should not be transformed to extend to full width when under elements like admonitions etc.  There may be more scenarios, but I'm not sure yet.

### Actual behavior

On mobile devices, it attempts to extend tabbed code blocks to full width stripping the border and making them look odd.

### Steps to reproduce the bug

1. View tabbed code under an admonition on a mobile device.

### Package versions

* Python: NA
* MkDocs: NA
* Material: 2.8.0

### Project configuration

NA

### System information

* OS: NA
* Browser: NA

**Comment by squidfunk** [comment]
I thought I catched all cases :D Thanks for reporting! It should be exactly like you wrote - only stretch to full width on the root level. I will provide a fix.

**Comment by squidfunk** [comment]
Fixed in 87f9a3a25bbfbb6b12e536cb637ea807fcf4203a

**Comment by squidfunk** [comment]
Released as part of 2.9.0
